src/FLEx-ChorusPluginTests/Infrastructure/Handling/ConfigLayout/DictionaryConfigurationHandlerStrategyTests.cs

```diff
@@ -23,33 +20,25 @@ namespace FLEx_ChorusPluginTests.Infrastructure.Handling.ConfigLayout
 	public class DictionaryConfigurationHandlerStrategyTests : BaseFieldWorksTypeHandlerTests
 	{
 		private TempFile _configFile;
-		private TemporaryFolder _tempFolder;
-		private string _xsdPathInProjRepo;
+		private string _xsdSourcePath;
 
 		[TestFixtureSetUp]
 		public override void FixtureSetup()
 		{
-			base.FixtureSetup();
 			// Copy the schema file to where the Strategy looks
 			var appsDir = Path.GetDirectoryName(Utilities.StripFilePrefix(Assembly.GetExecutingAssembly().CodeBase));
-			var xsdPath = Path.Combine(appsDir, "TestData", "Language Explorer", "Configuration", SharedConstants.DictConfigSchemaFilename);
-			_tempFolder = new TemporaryFolder("Temp");
-			_xsdPathInProjRepo = Path.Combine(_tempFolder.Path, SharedConstants.DictConfigSchemaFilename);
-			File.Copy(xsdPath, _xsdPathInProjRepo, true);
+			_xsdSourcePath = Path.Combine(appsDir, "TestData", "Language Explorer", "Configuration", SharedConstants.DictConfigSchemaFilename);
 		}
 
 		[TestFixtureTearDown]
-		public override void FixtureTearDown()
-		{
-			ClearCachedSchemaPathFromHandler();
-			_tempFolder.Dispose();
-			_tempFolder = null;
-			base.FixtureTearDown();
-		}
+		public override void FixtureTearDown() {} // mask base.FixtureTearDown (it is called by this.TestTearDown)
 
 		[SetUp]
 		public override void TestSetup()
 		{
+			// set up the fixture for each test, so each starts with a fresh HandlerStrategy, because DictionaryConfigurationHandlerStrategy
+			// caches the path to the schema, but some tests require the schema's presence and others require its absence.
+			base.FixtureSetup();
```

Comment:
Calling the base method of some other method is a pretty stinky code smell. I think I'll try my suggestions from the previous review to see if they work.